tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -257,6 +288,11 @@ module.exports = {
   getTaskState,
   openForm,
   formTitle,
+  sidebarFilterToDate,
+  sidebarFilterFromDate,
+  openSidebarFilter,
+  openSidebarFilterDateAccordion,
+  setSidebarFilterDate,
```

Comment:
Minor, but it would encapsulate some complexity within this util (instead of exposing it to the tests) if you were to export separate `setSidebarFilterFromDate` and `setSidebarFilterToDate` functions instead of exporting `sidebarFilterToDate`, `sidebarFilterFromDate`, and `setSidebarFilterDate`.  The new functions could still use those three internally, but the consuming test code does not need to know about them.